packages/devtools_app/lib/src/memory/memory_filter.dart

```diff
@@ -299,67 +333,75 @@ class SnapshotFilterState extends State<SnapshotFilterDialog>
     //      - one row of buttons Ok/Cancel
     // For very tall app keep the dialog at a reasonable height w/o too much vertical whitespace.
     // The listbox area is the area that grows to accommodate the list of known libraries.
-    // TODO(devoncarew): Convert to a DevToolsDialog.
-    return Dialog(
-      key: snapshotFilterDialogKey,
-      child: LayoutBuilder(
-        builder: (BuildContext context, BoxConstraints constraints) {
-          return Container(
-            width: MediaQuery.of(context).size.width / 3,
-            height: constraints.maxHeight < 400
-                ? constraints.maxHeight
-                : constraints.maxHeight * .3 + (400 * .7),
-            child: Padding(
-              padding: const EdgeInsets.only(left: 15),
-              child: Column(
-                crossAxisAlignment: CrossAxisAlignment.start,
+
+    final constraints = BoxConstraints(
+      maxWidth: defaultDialogWidth,
+      maxHeight: MediaQuery.of(context).size.height,
+    );
+
+    Widget subTitle(String title) => TextField(
+          decoration: InputDecoration(
+            border: InputBorder.none,
+            isDense: true,
+            labelText: title,
+          ),
+        );
+
+    return DevToolsDialog(
+      title: dialogTitleText(
+          Theme.of(context), 'Memory Filter Libraries and Classes'),
+      includeDivider: false,
+      content: Container(
+        width: defaultDialogWidth,
+        child: Padding(
+          padding: const EdgeInsets.only(left: 15),
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              Column(
                 children: [
-                  Column(
+                  subTitle('Filter the Snapshot:'),
+                  Row(
                     children: [
-                      const TextField(
-                        decoration: InputDecoration(
-                            border: InputBorder.none,
-                            isDense: true,
-                            labelText: 'Filter Snapshot'),
-                      ),
-                      Row(
-                        children: [
-                          NotifierCheckbox(
-                              notifier: controller.filterPrivateClasses),
-                          const Text('Hide Private Class e.g.,_className'),
-                        ],
+                      NotifierCheckbox(
+                        notifier: controller.filterPrivateClasses,
                       ),
-                      Row(
-                        children: [
-                          NotifierCheckbox(
-                              notifier: controller.filterZeroInstances),
-                          const Text('Hide Classes with No Instances'),
-                        ],
+                      const Tooltip(
```

Comment:
use DevToolsTooltip. Feel free to add the preferBelow param to that class and pass it through to the tooltip widget in the build method

Reply:
Done.